components/infra-proxy-service/storage/postgres/migration/sql/04_convert_table_to_human_readable_ids.up.sql

```diff
@@ -0,0 +1,23 @@
+-- drop foreign key constraint.
+ALTER TABLE orgs DROP CONSTRAINT orgs_server_id_fkey;
+
+-- remove UUID column data type.
+ALTER TABLE servers ALTER COLUMN id TYPE TEXT;
+ALTER TABLE orgs ALTER COLUMN id  TYPE TEXT;
+ALTER TABLE orgs ALTER COLUMN server_id TYPE TEXT;
+
+-- add foreign key constraint.
+ALTER TABLE orgs ADD CONSTRAINT orgs_server_id_fkey 
```

Comment:
can we get some idempotency clauses in here? (e.g. `if not exists`)

Reply:
Have added some of cautions statement, but `ADD CONSTRAINT` doesn't have any `IF NOT EXISTS` option. ref: https://www.postgresql.org/docs/9.6/sql-altertable.html